### PR TITLE
Add display_text to COMMAND DOCS

### PR DESCRIPTION
**Pull request by guybe7**
ref: https://github.com/redis/redis/pull/11051/

### CODE REVIEW

File: docs/reference/command-arguments.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ Every element in the _arguments_ array is a map with the following fields:
 * **name:** the argument's name, always present.
   The name of an argument is given for identification purposes alone.
   It isn't displayed during the command's syntax rendering.
```
Comment (itamarhaber) [inline docs/reference/command-arguments.md:18]: Is this correct? Isn't name the default display?

File: docs/reference/command-arguments.md [hunk 2]
```diff
@@ -16,6 +16,9 @@ Every element in the _arguments_ array is a map with the following fields:
...
+* **display_text:** the argument's display string, present in arguments that have
+  a displayable representation (all arguments which are not oneof/block)
+  This is the string used in the command's syntax rendering.
```
Comment (itamarhaber) [inline docs/reference/command-arguments.md:21]: Newline after period, minor edit

```suggestion
* **display_text:** the argument's display string, present in arguments that have  a displayable representation (all arguments that aren't oneof/block).
  This is the string used in the command's syntax rendering.
```

**Review by itamarhaber — APPROVED** [review]

File: docs/reference/command-arguments.md (hunk 1 above)
Comment (guybe7) [reply docs/reference/command-arguments.md:18]: no, display is the default display (it's just that if "display" is not present in the json, COMMANDS DOCS will use "name" for "display")

bottom, line, display is or display, name is for identification

File: docs/reference/command-arguments.md [hunk 3]
```diff
@@ -16,6 +16,8 @@ Every element in the _arguments_ array is a map with the following fields:
 * **name:** the argument's name, always present.
   The name of an argument is given for identification purposes alone.
   It isn't displayed during the command's syntax rendering.
+* **display_text:** the argument's display string, present in arguments that have  a displayable representation (all arguments that aren't oneof/block).
```
Comment (enjoy-binbin) [inline docs/reference/command-arguments.md:19]:
```suggestion
* **display_text:** the argument's display string, present in arguments that have a displayable representation (all arguments that aren't oneof/block).
```

File: docs/reference/command-arguments.md (hunk 1 above)
Comment (oranagra) [reply docs/reference/command-arguments.md:18]: so don't we still need to clear that out?
i.e. that the `name` should be used as display only if display_text is missing?

also, maybe we wanna mention something about the uniqueness of `name`?

Comment (guybe7) [reply docs/reference/command-arguments.md:18]: "display_text" is never missing in COMMAND DOCS (except for block/oneof, in which case there's nothing to display)

and yes, i'll add a few words on the uniqueness of `name`

Comment (guybe7) [reply docs/reference/command-arguments.md:18]: perhaps a piece of code is worth more than a thousand comments:
```
addReplyBulkCString(c, "display_text");
addReplyBulkCString(c, args[j].display_text ? args[j].display_text : args[j].name);
 ```

Comment (oranagra) [reply docs/reference/command-arguments.md:18]: ohh, i managed to forget we closed that loop in redis.

File: docs/reference/command-arguments.md [hunk 4]
```diff
@@ -16,6 +16,12 @@ Every element in the _arguments_ array is a map with the following fields:
 * **name:** the argument's name, always present.
   The name of an argument is given for identification purposes alone.
   It isn't displayed during the command's syntax rendering.
+  The same name can appear more than once in the entire argument tree,
+  but it is unique compared to other sibling arguments' names
+  This allows to attain a unique identifier for each argument (the concatination
+  of all names in the path from the root to any argument).
```
Comment (guybe7) [inline docs/reference/command-arguments.md:22]: @itamarhaber please rephrase if needed

**Review by oranagra — APPROVED** [review]

File: docs/reference/command-arguments.md [hunk 5]
```diff
@@ -16,6 +16,12 @@ Every element in the _arguments_ array is a map with the following fields:
 * **name:** the argument's name, always present.
   The name of an argument is given for identification purposes alone.
   It isn't displayed during the command's syntax rendering.
+  The same name can appear more than once in the entire argument tree,
+  but it is unique compared to other sibling arguments' names
+  This allows to attain a unique identifier for each argument (the concatenation
+  of all names in the path from the root to any argument).
```
Comment (itamarhaber) [inline docs/reference/command-arguments.md:22]:
```suggestion
  The same name can appear more than once in the entire argument tree, but it is unique compared to other sibling arguments' names.
  This allows obtaining a unique identifier for each argument (the concatenation of all names in the path from the root to any argument).
```

**Review by itamarhaber — APPROVED** [review]

**Pull request merged**